rfc-097-verify-specific-start-pages.md

```diff
@@ -0,0 +1,136 @@
+desirable as it may give service sign-in options that would direct the user away from Verify and the
+user's chosen IDP (for example, if the user chooses to login with Government Gateway).
+
+The approach described in this document is to implement alternate service start pages, that
+will display all important information to the user but only give them the option of using
+Verify and, hence, their chosen IDP. 
+
+### Considerations
+
+#### Discoverability
+
+Any customised content is designed to be reachable solely via hyperlinks from IDPs, which will redirect via the Verify hub.
+Therefore the new content MUST not be indexed and reachable by performing, for example, a Google search or GOV.UK search. 
+
+
+#### Content Schemas
+
+There are two GOV.UK content schemas used for service start pages. Most services use the `transaction` schema but some use the 
+`guide` schema. Any solution MUST work for, at least, both these content schemas.
+
+
+## Proposal
+
+### Short term/interim solution - Alternate Content
+
+In the short term, and in the interests of getting up and running quickly, we can simply take a copy of the existing content in
+GOV.UK's content publisher service, alter it, and publish it with a different URL slug (as specified in the `base_url` property 
+of the content item. A suggestion would be to append `-verify` to the the end of the existing slug.
+
+For example:
+```
+// Original Content Item
+{
+    "analytics_identifier": null,
+    "base_path": "/my-rp-transaction-page",
+    "content_id": "abcd1234-5678-90ab-cdef-0123456789ab",
+    ...
+}
+```
+
+```
+// Alternate Content Item
+{
+    "analytics_identifier": null,
+    "base_path": "/my-rp-transaction-page-verify",
+    "content_id": "abcd1234-5678-90ab-cdef-abcdef123456",
+    ...
+}
+```
+
+[**needs confirmation**] This solution works well for `transaction` content schema, and looks like it should work with `guide` although more care will 
+need to be taken the latter to ensure all slugs are updated and any explicit hyperlinks updated also.
+
+#### Searching
+ 
+[**for discussion**] The GOV.UK team state there is an existing method for ensuring that these alternate content pages are not searchable
+within the GOV.UK website, a change will need to be implemented on GOV.UK to ensure the these pages are not indexable by 
+search engine web crawlers. Even though the new alternate content will not be linked from anywhere, so would not be discoverable by crawlers, 
+GOV.UK does publish a `sitemap.xml`. We need to ensure these alternate content pages are not published in `sitemap.xml` and/or include the
+ `ROBOTS` meta tag with a `NOINDEX` value.
+ 
+### Long term solution - Content Variations
+
+A long term solution would be for GOV.UK to support 'variants' of content in the content schemas. This should allow a variant of
+the page to be requested in the URL, for example, `http://www.gov.uk/<base_url>/<variant-name>`
```

Comment:
Content on GOV.UK can already register routes for multiple paths. Rendering apps can then process this however they want.

A common example is the Travel Advice pages, these two pages relate to the same content item:
 - https://www.gov.uk/foreign-travel-advice/france
 - https://www.gov.uk/foreign-travel-advice/france/safety-and-security

I'm not sure how well this fits with what you're trying to do though...

Reply:
A version of this is how I envisage these start pages working in practice. Basically, get Mainstream Publisher to register the "variant" routes at the same time and route them to Government Frontend as usual where they will be picked up and rendered.

The part that needs work is storing the variant content somewhere in the original content item so Government Frontend can render it without too much custom/hardcoded logic (just enough to understand how to get and display the variant content).

Reply:
> The part that needs work is storing the variant content somewhere in the original content item so Government Frontend can render it without too much custom/hardcoded logic (just enough to understand how to get and display the variant content).

Well, crossing what's described here with what Government Frontend/Travel Advice Publisher do for travel advice pages seems sensible. So having something in the details, but maybe calling it variants rather than parts.